### PR TITLE
If custom linter is not found by absolute url or relative to cwd, check relative to .lesshintrc

### PR DESCRIPTION
**Pull request by WattyRev**
**Which issue, if any, does this resolve?**
#372 

**Is there anything in this PR that needs extra explaining or should something specific be focused on?**
Basic gist is that I am adding _path to the config data that contains the path to the .lesshintrc file. This allows me to find custom linters relative to .lesshintrc.

### CODE REVIEW

**Comment by WattyRev** [comment]
*bump*

File: lib/config-loader.js [hunk 1]
```diff
@@ -14,7 +14,9 @@ const loadConfig = function (path) {
         data = data.slice(1);
     }
 
-    return JSON.parse(data);
+    data = JSON.parse(data);
+    data._path = path;
```
Comment (shellscape) [inline lib/config-loader.js]: I'm personally not a fan of properties led with an underscore. I think `data.filePath` or `data.configPath` would be good and semantic here

Comment (WattyRev) [inline lib/config-loader.js]: Updated.

Comment (shellscape) [inline lib/config-loader.js]: cool. gotta let the others chime in before I can merge :)

**Comment by jwilsson** [comment]
@WattyRev Thanks for contributing! 🎉 

I'll see if I can get a new release out over the weekend.

**Pull request merged**